deployments/goerli/usdc/deploy.ts

```diff
@@ -24,32 +24,10 @@ async function deployContracts(deploymentManager: DeploymentManager, deploySpec:
   // Deploy governance contracts
   const { COMP, fauceteer, timelock } = await cloneGov(deploymentManager);
 
-  const usdcProxyAdmin = await deploymentManager.deploy('USDC:admin', 'vendor/proxy/transparent/ProxyAdmin.sol', []);
   const usdcImpl = await deploymentManager.clone('USDC:implementation', clone.usdcImpl, []);
   const usdcProxy = await deploymentManager.clone('USDC', clone.usdcProxy, [usdcImpl.address]);
-  const usdcProxyAdminSlot = '0x10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b';
   const USDC = usdcImpl.attach(usdcProxy.address);
```

Comment:
> const USDC = usdcImpl.attach(usdcProxy.address);

we should be able to delete this entirely, right? we're no longer cloning from mainnet USDC, we're adding an already-deployed USDC to `roots.json`